src/linuxforhealth/csvtofhir/model/contract.py

```diff
@@ -121,6 +121,12 @@ class FileDefinition(ImmutableModel):
                     "Required field when fileType = fixed width"
     )
 
+    pandas_params: Optional[Dict[str, Any]] = Field(
```

Comment:
The reason I went with a generic/open ended solution here is based on the number of parameters supported by the read_csv and read_fwf functions. Maintaining all of those as explicit parameters would pollute the data contract model quite a bit, and I felt like this was the most flexible solution.

In addition I feel like the users of this project will be testing their data-contract files against representative sample data while developing the mappings, and any panda parameters that they have decided to use should be tested at the same time.

Having said all that, the only parameter we need right now is `skiprows`, so if the maintainers feel like that is the way to go, I can only add support for that specific parameter, and we can add more as and when needed

Reply:
@hammadk373 - thank you for the thoughtful response. While I do favor having "specific" parameters I can certainly understand the other point of view. Since no one else has weighed in, what would you think about the following:

- move forward with the FileDefinition.pandas_params field
- update https://github.com/LinuxForHealth/CsvToFHIR/blob/main/docs/datacontract.md with information regarding the pandas_params field

Additionally, I would suggest that we create a new issue to migrate the "pandas" related settings in General and FileDefinition to the pandas_params fields, to ensure that we source "pandas" settings from a single location. In the near term this would require a DataContract change, but in the long term I think it would be useful as it would be consistent. 

@klwhaley , @LisaWellman, and @cragun47  we can chat about this last point and it's affect on our internal apps and services.

Reply:
I have updated the data contract to include the new `pandas_params` field.